libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -35,6 +35,8 @@ export class Function extends cloud.FunctionBase {
   private policyStatements?: any[];
   /** Function ARN */
   public readonly arn: string;
+  /** Function Qualified ARN */
```

Comment:
Why not always return this instead of ARN?

If the "safe" version is the qualified arn, then we should only use it

Reply:
@eladb @raywonkari  I just noticed this was a discussion 😅 I made a comment about this below. I think what this will end up doing is binding the IAM permissions to a specific version which means that every function change will also deploy a new IAM policy update. Which if we want to limit the policy to only allow the function to invoke the latest version we could do that with an alias that would not require a new policy change per deploy.

It may be worth thinking about exposing both arn maybe?

Reply:
So maybe we should add a `unqualifiedArn` or something like that, so `arn` will point to the qualified one which is the current version (which I am assuming is what users usually want), and if you want to point to any version, you can use the `unqualifiedArn`

(design tenet: best practice should be the path of least resistance).